assets/js/base/components/select/style.scss

```diff
@@ -90,6 +90,7 @@
 		margin: 0;
 		max-height: 300px;
 		overflow: auto;
+		color: #000;
```

Comment:
I wonder if this color should be `$input-text-active`. Otherwise, the text color of options look a bit darker than the selected value:

![imatge](https://user-images.githubusercontent.com/3616980/103767112-56c72780-5020-11eb-90ac-50b68a1559dc.png)